charts/alfresco-process-services/Chart.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v2
+description: A Helm chart for Alfresco Process Services
+name: alfresco-process-services
+version: 0.11.0-alpha.0
+appVersion: 24.1.0
+maintainers:
+  - name: Alfresco
```

Comment:
Please update to `Hyland` or remove it (looks like we're not setting this in others charts)